packages/extension/src/companion/App.tsx

```diff
@@ -41,6 +44,10 @@ export type CompanionData = { url: string; deviceId: string } & Pick<
 
 const refreshTokenKey = 'refresh_token';
 
+const companionModals = {
+  [LazyModal.UpvotedPopup]: UpvotedPopupModal,
+};
```

Comment:
I don't think we should start doing this, basically companion needs to support all modals down the line.